app/models/state_file_base_intake.rb

```diff
@@ -93,6 +93,15 @@ def self.selected_intakes_for_deadline_reminder_notifications
       .select(&:should_be_sent_reminder?)
   end
 
+  def self.selected_intakes_for_first_deadline_reminder_notification
+    intakes = left_joins(:efile_submissions)
```

Comment:
i don’t think you need to join on the efile submissions here since you’re not using it in a query